opimodel/rules.py

```diff
@@ -21,3 +34,25 @@ def __init__(self, prop_id, pv, threshold):
         super(GreaterThanRule, self).__init__(prop_id)
         self._pv = pv
         self._threshold = threshold
+
+
+class SelectionRule(Rule):
+
+    def __init__(self, prop_id, pv, options, var="pv0"):
+        """ Simple selection rule, e.g.:
+
+            widget.rules = []
+            options = [(-1, colors.INVALID), (1, colors.MAJOR), (2, colors.MINOR)]
```

Comment:
Perhaps. The list ensures the order in which the cases are written, but that's not important for the equality comparisons we have implemented.